### ISSUE TITLE
Possible to pass the relative into the object?

### ISSUE THREAD

**Issue by corydorning** [VFK/gulp-html-replace#9]
I have several HTML files i'm looping through in order to update my 'source' files with my 'dist' files. An example would be `/indext.hml` linking to `<script src="js/main.js>` and `/pages/foo.html` linking to the same file via `<script src="../js/main.js">`.

Given the relative paths, setting a static option value doesn't work. I could set a root value, but this is actually a boilerplate build system that is shared, so that makes it a tougher sell. Here's my task, didn't know if you had any ideas on how it would be possible to pass the base path in...

```
var gulp = require('gulp'),
    htmlreplace = require('gulp-html-replace'),
    htmlFiles = ['./**/*.html'];

gulp.task('html', function() {
    return gulp
      .src(htmlFiles , { cwd: './src', base: './src' })
      .pipe(htmlreplace({
        'css-vendor': relative.path  + 'vendor.css',
        'css-app': relative.path + 'app.css',
        'js-vendor': relative.path + 'vendor.js',
        'js-app': relative.path + 'app.js'
      }))
      .pipe(gulp.dest('./dist'));
```

**Comment by corydorning** [comment]
so I'm back to this and unfortunately, I can't seem to get `resolvePaths` to work. here's what i have:

```
return gulp.src(paths.app.html.files, { cwd: paths.root.src)

        // optimize: replace css/js paths in index.html
        .pipe(_gulp.htmlReplace({
            'AppJS': paths.app.js.dir + paths.app.js.name
          }, {
            resolvePaths: true
          }
         ))
        .pipe(gulp.dest(paths.root.www))
```

This however, turns this `<script src="js/app.js"></script>` into this `<script src="..\js\app.js"></script>` which is right sans the backslash.

**Comment by VFK** [comment]
@corydorning i'm sorry for the delay, i was away for some time. So, the path is correct, the slashes are wrong, right? I fixed slashes in 1.4.5, please check it out.
